pallets/xyk/src/lib.rs

```diff
@@ -546,6 +552,9 @@ pub mod pallet {
 				<PoolAssets<T>>::remove(&pair_account);
 				<TotalLiquidity<T>>::remove(&pair_account);
 
+				// Ignore the failure, this cant stop liquidity removal
+				let _ = T::NonDustableWhitelistHandler::remove_account(&pair_account);
```

Comment:
I wonder whether we might want to log this. If this happens something went very wrong.

Reply:
added error log in case of error.